stories/toast/props-table.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { ApiTable } from "../storybook-common/api-table";
+import { ApiTableSectionProps } from "../storybook-common/api-table/types";
+
+const DATA: ApiTableSectionProps[] = [
+    {
+        attributes: [
+            {
+                name: "type",
+                mandatory: true,
+                description: (
+                    <>
+                        Type of <code>Toast</code>
+                    </>
+                ),
+                propTypes: [`success`, `warning`, `error`, `info`],
```

Comment:
missing double quotes to distinguish these as string constants

```suggestion
                propTypes: [`"success"`, `"warning"`, `"error"`, `"info"`],
```